File: API/HandsOn/MVCClient/Models/Product.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace MVCClient.Models
{
    public class Product
    {
       
        public int ProductId { get; set; }
        public string? Name { get; set; } 
        public int Price { get; set; } 
        public int? Stock { get; set; } 
    }
}

File: React/HandsOn/react-functional-components/src/Product.js
const Product = (props) => {
  return (
    <div>
      <pre>
        Name:{props.item.name}
        Price:{props.item.price}
        Stock:{props.item.stock}
      </pre>
    </div>
  );
};
const Shop = () => {
  const product = { name: "Iphone", price: 134000, stock: 3 };
  return (
    <>
      <Product item={product} />
      <Product item={{ name: "Bottle", price: 560, stock: 20 }} />
    </>
  );
};
export default Shop;
